library/dd-sdk-android-session-replay/src/main/kotlin/com/datadog/android/sessionreplay/processor/SnapshotProcessor.kt

```diff
@@ -36,11 +36,6 @@ internal class SnapshotProcessor(
 
     @MainThread
     override fun process(node: Node, orientationChanged: OrientationChanged?) {
-        if (node.wireframes.isEmpty()) {
```

Comment:
Nope, before I was keeping a single element `List<Wireframe>` just in case I need to add more in there. That case should never have happened even before but just because it was a `List` I had to add the test. Right now because we have a non-optional `Wireframe` property there this test will not be needed.